Automattic/wp-calypso
# Hosting Command Palette: enable the feature on staging

Related to https://github.com/Automattic/dotcom-forge/issues/4524

## Proposed Changes

This PR enables the command palette on `/sites` page on the staging environment. 

## Testing Instructions

* Look through the code and confirm that `yolo/command-pallette` is set to true on staging. 

## Pre-merge Checklist
